src/basic/cgroup-util.h

```diff
@@ -115,12 +115,24 @@ extern const uint64_t cgroup_io_limit_defaults[_CGROUP_IO_LIMIT_TYPE_MAX];
 const char* cgroup_io_limit_type_to_string(CGroupIOLimitType t) _const_;
 CGroupIOLimitType cgroup_io_limit_type_from_string(const char *s) _pure_;
 
+/* Special values for the cpu.idle attribute */
+#define CGROUP_CPU_IDLE_INVALID INT64_MAX
+#define CGROUP_CPU_IDLE_DEFAULT INT64_C(0)
+#define CGROUP_CPU_IDLE_MIN INT32_MIN
+#define CGROUP_CPU_IDLE_MAX INT32_MAX
```

Comment:
> This seems iffy. Why allow signed values? Shouldn't this be `uint64_t` ?

this is because `cpu.idle` is [s64](https://github.com/torvalds/linux/commit/304000390f88d049c85e9a0958ac5567f38816ee#diff-cc1a82129952a910fdc4292448c2a097a2ba538bebefcf3c06381e45639ae73eR10371) in the kernel, maybe there will be more values like `-1` in the future, I think `int64_t` can make more compatibility